contracts/azorius/LinearERC20Voting.sol

```diff
@@ -202,7 +202,7 @@ contract LinearERC20Voting is BaseStrategy, BaseQuorumPercent, BaseVotingBasisPe
      * @param _proposalId id of the Proposal
      * @return uint256 the address' voting weight
      */
-    function getVotingWeight(address _voter, uint32 _proposalId) public view returns (uint256) {
+    function getVotingWeight(address _voter, uint32 _proposalId) public view virtual returns (uint256) {
```

Comment:
since this would require opening this up to virtual, it would be great to iron this out before we deploy the Azorius contracts to mainnet.

Reply:
For the record I'm fairly certain that the `virtual` keyword is simply a compile-time sanity check, doesn't affect the compiled bytecode either way.

Reply:
oh cool, then all good on timing

Reply:
Would be best to double check my assumption here though 😅

Reply:
I do think you need the `virtual` keyword to be able to override the function on an inheriting contract: 

https://docs.soliditylang.org/en/v0.8.20/contracts.html#function-overriding